site/guide/4-requests.md

```diff
@@ -678,7 +678,12 @@ struct Task<'r> {
 fn new(task: Json<Task<'_>>) { /* .. */ }
 ```
 
-See the [JSON example](@example/serialization/src/json.rs) on GitHub for a complete example.
+! NOTE: Json support requires the `json` feature flag to be enabled in Rocket
```

Comment:
Could we use proper casing for "JSON" when used in a sentence?

```suggestion
! NOTE: JSON support requires the `json` feature flag to be enabled in Rocket
```

Reply:
I've seen both, but I guess we should be consistent. Thanks for pointing this out.